src/hooks/useSendMessage.ts

```diff
@@ -96,8 +82,25 @@ function useSendMessage({ isRunning, isValidCall, setIsRunning, input, type, wei
         options.signer = injector.signer;
         sourceAccount = account.address;
       }
-
       const unsub = await bridgeMessage.signAndSend(sourceAccount, { ...options }, ({ events = [], status }) => {
+        if (startProcess) {
```

Comment:
i would like to know if there is any internal state in the callback similar as `status.isInBlock` to check if the transaction was signed. In case not , i'm fine with this ( i does the job and solves the bug )

Reply:
I think there is `status.isReady || status.isFuture` that can be used for that.

Also it seems we are not handling an error scenario at all here (netiher pool inclusion nor 3rd argument to the callback, namely `dispatchError`), but I guess that's separate issue.